src/styles/header.css

```diff
@@ -1,3 +1,71 @@
+/* header */
+
 .ui.header {
   font-family: var(--gotham);
 }
+
+h1.ui.header {
+  font-size: var(--h1);
+  line-height: var(--h1-height);
+  font-weight: var(--black-weight);
+}
+
+h2.ui.header {
+  font-size: var(--h2);
+  line-height: var(--h2-height);
+  font-weight: var(--black-weight);
+}
+
+h3.ui.header {
+  font-size: var(--h3);
+  line-height: var(--h3-height);
+  font-weight: var(--black-weight);
+}
+
+h4.ui.header {
+  font-size: var(--h4);
+  line-height: var(--h4-height);
+  font-weight: var(--black-weight);
+}
+
+h5.ui.header {
+  font-size: var(--h5);
+  line-height: var(--h5-height);
+  font-weight: var(--black-weight);
+}
+
+h6.ui.header {
+  font-size: var(--subheading);
+  line-height: var(--subheading-height);
+  font-weight: var(--book-weight);
```

Comment:
@Dart-EY This is where the font weight variables get a little tricky. It turns out that not all headings use the same weight, so `--weight-heading` won't be sufficient. I guess we could introduce `--weight-subheading` but that's a little ambiguous. It's probably easier if we just write `400` directly.

Reply:
Got it. I will make the changes to the input PR.